perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -308,27 +310,100 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			ac.stateChangingCheck();
 
 			ac.getGroupsManager().addMember(ac.getSession(),
-					ac.getGroupById(parms.readInt("group")),
-					ac.getMemberById(parms.readInt("member")));
+				ac.getGroupById(parms.readInt("group")),
+				ac.getMemberById(parms.readInt("member")));
 			return null;
 		}
 	},
 
+	/*#
+	 * Removes members from a group.
+	 *
+	 * @throw MemberNotExistsException When member doesn't exist
+	 * @throw NotGroupMemberException  When member is not in the group
+	 * @throw GroupNotExistsException When the group doesn't exist
+	 * @throw WrongAttributeAssignmentException When assigning atribute to wrong entity
+	 * @throw AttributeNotExistsException When attribute doesn't exist
+	 * @throw ExternallyManagedException When the group is externally managed
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param members List<Integer> Members - can be empty <code>id</code>
+	 */
+	removeMembers {
+		@Override
+		public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
+			ac.stateChangingCheck();
+			List<Integer> memberInts = parms.readList("members", Integer.class);
+			if (memberInts == null) {
+				throw new RpcException(RpcException.Type.MISSING_VALUE, "parameter 'members' not sent");
+			}
+			Group group = ac.getGroupById(parms.readInt("group"));
+			List<Member> members = new ArrayList<>();
+			for (Integer memberInt : memberInts) {
+				members.add(ac.getGroupsManager().getGroupMemberById(ac.getSession(), group, memberInt));
```

Comment:
You should try/catch this call and silently ignore ``MemberNotExistsException`` and ``NotGroupMemberException``.